typescript/sdk/src/ism/types.ts

```diff
@@ -48,8 +51,14 @@ export type AggregationIsmConfig = {
   threshold: number;
 };
 
+export type OpStackIsmConfig = {
+  type: ModuleType.OP_STACK;
+  nativeBridge: Address;
```

Comment:
I thought this authenticated the hook address

Reply:
added setAuthorizedHook to the end of deployOpStack(), the ismConfig doesn't need a hook address